### PR TITLE
fix(demos): Too much space between toolbar title and left icon.

### PR DESCRIPTION
**Pull request by touficbatache**
There's too much space between the toolbar title and left icon. This PR fixes it.

demo

### CODE REVIEW

**Comment by touficbatache** [comment]
I didn't remove it @williamernest take a look at the code.

File: demos/common.scss [hunk 1]
```diff
@@ -79,7 +79,6 @@ fieldset {
 
 .catalog-logo,
 .catalog-back {
-  padding-right: 24px;
```
Comment (williamernest) [inline demos/common.scss]: We should keep the `padding-right` property for the catalog-logo class.

File: demos/button.html [hunk 2]
```diff
@@ -74,9 +74,7 @@
     <header class="mdc-toolbar mdc-toolbar--fixed">
       <div class="mdc-toolbar__row">
         <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-          <span class="catalog-back">
```
Comment (williamernest) [inline demos/button.html]: We shouldn't be removing this span to change the `padding-right`.

Comment (touficbatache) [reply demos/button.html]: Why not removing the `span`? I kept the class :smiley:

File: demos/common.scss (hunk 1 above)
Comment (touficbatache) [reply demos/common.scss]: I don't think we should... There is too much space between the logo and the title...

Comment (touficbatache) [reply demos/common.scss]: Ah wait, there is already a `padding-right:24px;` applied to the remaining class

Comment (williamernest) [reply demos/common.scss]: Yup, not needed anymore.

Comment (touficbatache) [reply demos/common.scss]: That's why I removed it :smiley:

File: demos/button.html (hunk 2 above)
Comment (williamernest) [reply demos/button.html]: I guess it's not needed anymore :)

File: demos/common.scss (hunk 1 above)
Comment (williamernest) [reply demos/common.scss]: Yup, no longer needed.

**Pull request merged**